server/block/ladder.go

```diff
@@ -17,12 +17,12 @@ type Ladder struct {
 	sourceWaterDisplacer
 
 	// Facing is the side of the block the ladder is currently attached to.
```

Comment:
I'd probably leave a comment saying cube.FaceUp and cube.FaceDown do not do anything. Do just check in game first that it is the case

Reply:
in game the block appears but it is invisible and somehow you can still climb it